### ISSUE TITLE
the img in the assest

### ISSUE THREAD

**Issue by enasskmal** [lotus-1/correncyNN#17]
Well done girls did a stunning and useful application :)
You could add your img background to the file assets and use it.
This is to prevent your site to break if the image you are using disappears from the internet for some reason.

**Comment by NaremanPro** [comment]
you are right, we tried to do that but we couldn't use it as .png picture.. i will try to do my best :wink:
